aep/general/0231/aep.md.j2

```diff
@@ -9,25 +9,40 @@ provides this functionality.
 APIs **may** support batch get to retrieve a consistent set of resources.
 
 - The method's name **must** begin with `BatchGet`. The remainder of the method
-  name **should** be the plural form of the resource being retrieved.
+  name **must** be the plural form of the resource being retrieved.
 - The HTTP verb **must** be `GET`.
-- The HTTP URI **must** end with `:batchGet`.
-- The URI path **should** represent the collection for the resource, matching
-  the collection used for simple CRUD operations. If the operation spans
-  parents, a dash (`-`) **may** be accepted as a wildcard.
+- The HTTP URI **must** end with `:BatchGet`.
+- The URI path **must** represent the collection for the resource, matching the
+  collection used for simple CRUD operations. If the operation spans parents, a
+  [wilcard](./reading-across-collections) **may** be accepted.
 - There **must not** be a request body.
-  - If a GET request contains a body, the body **must** be ignored, and **must not** cause an error.
-- The operation **must** be atomic: it **must** fail for all resources or
-  succeed for all resources (no partial success). For situations requiring
-  partial failures, `List` (AEP-132) methods **should** be used.
-  - If the operation covers multiple locations and at least one location is
-    down, the operation **must** fail.
+  - If a GET request contains a body, the body **must** be ignored, and **must
+    not** cause an error.
 
 ### Request
 
 The request for a batch get method **should** be specified with the following
 pattern:
 
+- The request **must** include an array parameter which accepts the resource
+  paths specifying the resources to retrieve. The parameter **should** be named
+  `paths`.
```

Comment:
```suggestion
- The request **must** include an array field which accepts the resource
  paths specifying the resources to retrieve. The field **should** be named
  `paths`.
```

I think we use "field" reasonably consistently across the other AEPs.  (Suggestion applies to below uses of "parameter" as well.)

Reply:
Fine, but this is just going to make it clear that the AEPs are really about gRPC and HTTP is kinda an afterthought.